src/security_utilities_rust/src/microsoft_security_utilities_core/marvin.rs

```diff
@@ -68,13 +68,13 @@ pub fn compute_hash(data: &[u8], seed: u64, offset: i32, mut length: i32) -> i64
         1 => p0 = p0.wrapping_add(0x8000 | (data[remaining_data_offset as usize] as u32)),
         2 => {
             let d1 = (data[(remaining_data_offset as usize) + 1] as u32) << 8;
-            let d0: u32 = data[(remaining_data_offset as usize)] as u32;
+            let d0: u32 = data[remaining_data_offset as usize] as u32;
```

Comment:
>remaining_data_offset [](http://example.com/codeflow?start=31&length=21)

thanks for ongoing help clearing warnings!

Reply:
@michaelcfanning Thanks! I am going to burn the set of build warnings down to 0 in a subsequent pass.